src/jti_background.js

```diff
@@ -61,6 +64,35 @@ function saveTemplates (templateJSON, callback, responseData = null) {
     });
 }
 
+function getDomains (callback) {
+    var domainArray = {};
+    // Get the default domains
+    var domainArray2 = $.map(DefaultDomainList, function (domain, index) {
+        domain.default = true;
+        return domain;
+    });
+    // sort the default domains
+    domainArray2 = utils.sortArrayByProperty(domainArray2, 'name');
+    // Get the custom domains
+    chrome.storage.sync.get(StorageID, function (data) {
+        if (data[StorageID]) {
+            var domains = data[StorageID].options.domains;
+            domainArray = $.map(domains, function (domain, index) {
+                domain.default = false;
```

Comment:
It is an option that would work, and I could do it. However, I personally prefer explicit over implicit coding. I also think I'll still have to do the mapping anyway (mapping the domain object to the array), so I don't see a drawback to leaving this back.